cypress/integration/Signup.feature

```diff
@@ -37,8 +37,8 @@ Feature: Signup
         Then I see "There is a problem" in the page text
 
         # todo - note we should be selecting Mr like we do in the Valid scenario, not typing it in here, but due to system bug, after a previous error we get a text box
-        # instead of a dropdown. The line below will therefore need to change to When I select Mr on old style id, once that bug is fixed
-        When I type "Mr" into old style id "#name-title"
+        # instead of a dropdown. The line below will therefore need to change to When I select Mr on name-title, once this bug is fixed
```

Comment:
Is there a bug raised for this @nickdavis2001 ?

Reply:
Yes, did when previously encountered it